plugins/GSdx/GSdx.cpp

```diff
@@ -331,15 +331,15 @@ void GSdxApp::Init()
 	m_default_configuration["debug_glsl_shader"]                          = "0";
 	m_default_configuration["debug_opengl"]                               = "0";
 	m_default_configuration["disable_hw_gl_draw"]                         = "0";
-	m_default_configuration["dithering_ps2"]                              = "2";
+	m_default_configuration["dithering_ps2"]                              = "1";
```

Comment:
It will be Scaled default now, you need to change it in Renderer code too.

Reply:
I've reverted this to only use context menu.